presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    extra_hosts:
+       - "ad-test.presto.testldap.com:10.25.171.180"
```

Comment:
I'm also not sure if FB will merge a change that has a private IP hardcoded, so it might be this whole line will have to disappear and we won't be able to merge this before we have an AD docker container configured. Which shouldn't be that hard, there are plenty:  https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=ldap&starCount=0